app/models/commits_fetcher.rb

```diff
@@ -9,7 +9,7 @@ def initialize(repo, user, round)
   end
 
   def fetch(type = :daily)
-    user.gh_client.repos.branches(repo.owner, repo.name, {auto_pagination: true}).each do |branch|
+    user.gh_client.repos.branches(user: user.github_handle, repo: repo.name).list.each do |branch|
```

Comment:
Why did you change repo.owner to user.github_handle?

Reply:
It is easy to track.